scrapy_zyte_api/handler.py

```diff
@@ -82,7 +82,7 @@ def _prepare_api_params(self, request: Request) -> Optional[dict]:
                 f"provided as dictionary, got {type(request.meta.get('zyte_api'))} "
                 f"instead ({request.url})."
             )
-            raise IgnoreRequest()
+            raise
```

Comment:
It seems straightforward to cover it with a test as well - would you mind doing it, to make coverage check happy? Looks like it's a matter of adding another case for test_exceptions. If that's more than that, that's fine not to do it in this PR.

Reply:
I missed the codecov email on this. Thanks for raising! Addressed it on https://github.com/scrapy-plugins/scrapy-zyte-api/pull/42/commits/c94d8ea3576e9274767f5fdffd5621405c85c4e7